packages/newsroom-signup/src/NewsroomProfile/LearnMoreButton.tsx

```diff
@@ -78,45 +78,78 @@ export class LearnMoreButton extends React.Component<LearnMoreButtonProps, Learn
+              Civil tokens (CVL) enable the "community-owned and operated" Civil model to work. A token is a piece of
+              software, it’s not a physical coin. But it does have value and represents a piece of ownership in the
+              network.
             </Paragraph>
+            <Paragraph>
+              For those that want to get more technical, CVL is a utility token based on the ERC20 protocol. It’s a
+              value stored in a decentralized database that’s managed by Civil’s smart contracts, which allows the Civil
+              platform to interact with the Civil token. CVL is the software that bridges Civil with the Ethereum
+              blockchain.
+            </Paragraph>
+
+            <SectionHeader>Ether (ETH)</SectionHeader>
+            <Paragraph>
+              Ether (ETH) is different cryptocurrency than CVL but you will need some to to pay for certain actions on
+              Civil like sending tokens. That takes a small amount of computing effort, called Gas, which must be paid
+              for in ETH. When sending a transaction, you set a gas limit. You must set this limit carefully, because if
+              you set it too low and the transaction fails as a result, you will lose the gas / ETH that you spent.
+            </Paragraph>
+
             <SectionHeader>Applying to the Civil Registry</SectionHeader>
             <Paragraph>
-              You will need to deposit 0,000 CVL token with your application. This is to state the seriousness of your
-              intent to the community.
+              You will need to deposit $1,000.00 worth of CVL tokens with your application. This is to signal the
+              seriousness of your intent to the community.
             </Paragraph>
             <Paragraph>
-              Your application for the Newsroom will be up for community vote and review on the Application in Review
-              tab of the Civil Registry for 14 days. A Newsroom will be approved if there are no challenges.
+              Your application for the Newsroom will be up for community review on the Application in Review tab of the
+              Civil Registry for 14 days. A Newsroom will be approved if there are no challenges.
             </Paragraph>
             <Paragraph>
-              If the application is challenged by the community, there will be a 10 day period for the community to
-              Commit their votes. In order to finalize their votes, they have a 7 day period to Confirm. The newsroom
-              can request an Appeal from the Civil Council within 3 days. The Civil Council will decide on the appeal
-              within 14 days, and the challenge can be appealed, and the Commit and Confirm process begins again.
+              When a new Newsroom applies to be on the Civil Registry, the community has 14 days to review the
+              application. During the period, any CVL token holder may challenge this Newsroom if they believe the
```

Comment:
any CVL token holder sounds scarier then is true.  This sentence should include that they have to match the same $1000 dollar deposit the newsroom put down.

"any CVL token holder may challenge this newsroom by matching the application deposit if they believe the newsrooms ..."

Reply:
okay, looks good